src/app/shell/refresh.tsx

```diff
@@ -36,7 +39,7 @@ export default class Refresh extends React.Component<{}, { active: boolean }> {
     const { active } = this.state;
 
     return (
-      <span className="link" onClick={refresh} title={t('Header.Refresh')}>
+      <a className="link" onClick={refresh} title={t('Header.Refresh')} role="button">
```

Comment:
Hmm, could this stay a span with the `role="dialog"`? Making it an `a` tag makes it so that it can be right clicked and opened in a new tab, unless it helps with something else?

Reply:
I think I could make it show up correctly with `role="button"` and `tabindex="0"` to make it focusable, but the link gets me all that at once. I checked it out in VoiceOver. I could also use an actual `<button>` but I'd need to unstyle it.